cordova-sdk/package.json

```diff
@@ -25,6 +25,7 @@
     ]
   },
   "devDependencies": {
+    "@optimove-inc/cordova-sdk": "file:../cordova-sdk",
```

Comment:
What's the meaning to depend on the SDK in the SDK? (would expect in the sample app, but here seems strange, maybe it's correct, just don't understand)

Reply:
ofc it's not correct, removed 😬